source/apiVolontaria/volunteer/views.py

```diff
@@ -339,3 +339,15 @@ class ParticipationsId(generics.RetrieveUpdateDestroyAPIView):
 
     def get_queryset(self):
         return models.Participation.objects.filter()
+
+    def delete(self, request, *args, **kwargs):
+        participation = self.get_object()
+        if not participation.event.is_started:
+            return self.destroy(request, *args, **kwargs)
+        else:
+            content = {
+                'detail': "You can't delete a participation if the "
```

Comment:
We should wrap our custom error messages with `django.utils.translation.ugettext_lazy` to help in the traduction effort.